crates/ef-testing/src/models/case.rs

```diff
@@ -175,13 +179,34 @@ impl BlockchainTestCase {
                     )));
                 }
             }
-            let actual = sequencer.get_nonce(address)?;
+            // Nonce
+            let actual = sequencer.get_nonce_at(address)?;
             if actual != expected_state.nonce.0 {
                 return Err(RunnerError::Other(format!(
                     "nonce mismatch for {:#20x}: expected {:#32x}, got {:#32x}",
                     address, expected_state.nonce.0, actual
                 )));
             }
+            // Bytecode
+            let actual = sequencer.get_code_at(address)?;
+            if actual != expected_state.code {
+                return Err(RunnerError::Other(format!(
+                    "code mismatch for {:#20x}: expected {:#x}, got {:#x}",
+                    address, expected_state.code, actual
+                )));
+            }
+            // Skip sender address because of the difference in gas cost
```

Comment:
can't we force gas price = 0 instead?

Reply:
I can force gas price to 0 in the Kakarot sequencer but the ef test will still include gas cost in the evm, meaning I will always have a difference.

Reply:
indeed, and then we update the test definition for the gas_price only